packages/nimble-components/src/table/tests/table-keyboard-navigation.spec.ts

```diff
@@ -1245,7 +1245,7 @@ describe('Table keyboard navigation', () => {
             name: interactiveColumnName
```

Comment:
I thought when we wrote the keyboard navigation tests we had a vision of parameterizing them so they could easily cover new column types that had interesting keyboard navigation behavior. But I don't remember the details and in a quick scan I don't see the place where we'd expect to do that. @msmithNI do you expect these tests should be changing more significantly for the new column type?

Reply:
I ended up revamping the tests more based on feedback from Milan. Most of the tests I had for interactive column types were generic, and didn't really need to be run for each additional interactive column type. Those are the tests in this file now (which don't use the anchor column anymore, they use a test column type).

There's a handful of tests in [table-column-anchor.spec.ts](https://github.com/ni/nimble/blob/0f03ba3f181a96a585bb01aaf7da3ce74f6aac22/packages/nimble-components/src/table-column/anchor/tests/table-column-anchor.spec.ts#L650) that cover focusing the control via Tab, and blurring it via Esc, that would probably be good to add for this column type too. Others:
- Checking `tabbableChildren` in various configurations (examples also in the anchor test file)
- Any additional interactions specific to the column type (something with the menu open for this column type, potentially)

Reply:
Thanks for the suggestions. I've added quite a few additional tests following the patterns in the anchor column tests.